db_test.go

```diff
@@ -238,16 +234,9 @@ func TestOpen_ReadPageSize_FromMeta1_OS(t *testing.T) {
 	}
 
 	// Reopen data file.
-	if db, err := bolt.Open(path, 0666, nil); err != nil {
-		t.Fatalf("unexpected error: %s", err)
-	} else {
-		if db.Info().PageSize != os.Getpagesize() {
-			t.Fatalf("The page size is expected to be %d, but actually is %d", os.Getpagesize(), db.Info().PageSize)
-		}
-		if err := db.Close(); err != nil {
-			panic(err)
-		}
-	}
+	// Reopen data file.
+	db = btesting.MustOpenDBWithOption(t, path, nil)
+	require.Equalf(t, os.Getpagesize(), db.Info().PageSize, "The page size is expected to be %d, but actually is %d", os.Getpagesize(), db.Info().PageSize)
```

Comment:
I think the comment should be simpler like `check page size` because the require.Equal will show the actual and expected values.

Reply:
makes sense & update.